### PR TITLE
[nemo-qml-plugin-contextkit] Add license file. JB#55991

### PR DESCRIPTION
**Pull request by pvuorela**
The existing LICENSE states lgplv2+ so adjust the .spec accordingly too.

@Tomin1 @mlehtima

### CODE REVIEW

**Review by Tomin1 — APPROVED** [review]
I don't think there is any need to keep that LICENSE file as there is now full text too and files state which license is used anyway. But fine by me with or without it.

File: rpm/nemo-qml-plugin-contextkit-qt5.spec [hunk 1]
```diff
@@ -2,8 +2,8 @@ Summary: ContextKit QML bindings
 Name: nemo-qml-plugin-contextkit-qt5
 Version: 2.0
 Release: 2
```
Comment (Tomin1) [inline rpm/nemo-qml-plugin-contextkit-qt5.spec:4]: Hmm, though, I think this should be 0 or 1. Probably doesn't matter for OBS building at all. :thinking:

Comment (pvuorela) [reply rpm/nemo-qml-plugin-contextkit-qt5.spec:4]: OBS replaces that with something so doesn't matter, though most commonly it's just 1 here.

**Comment by pvuorela** [comment]
Kept the old license file too because it states the "or later" part of lgpl.

**Pull request merged**